astroquery/splatalogue/build_species_table.py

```diff
@@ -67,6 +67,7 @@ def get_json_species_ids(*, outfile='splat-species.json', base_url=conf.base_url
             species[kid['class'][0]][kid['value']] = kid.text
 
     with open(data_path(outfile), 'w') as f:
+        assert isinstance(species, dict)
```

Comment:
why the assert in the code? This might be skipped in practice, etc.

But would be nice to have test coverage

Reply:
can remove this.  I added it as part of debugging, doesn't really belong.